diff --git a/pkg/android/phoenix/build.gradle b/pkg/android/phoenix/build.gradle
index 03168ab..ac78368 100644
--- a/pkg/android/phoenix/build.gradle
+++ b/pkg/android/phoenix/build.gradle
@@ -1,6 +1,7 @@
 buildscript {
   repositories {
     google()
+    mavenCentral()
     jcenter()
   }
 
@@ -12,6 +13,7 @@ buildscript {
 allprojects {
     repositories {
         google()
+        mavenCentral()
         jcenter()
     }
 
